Automattic/wp-calypso
# Domain Search: Privacy Protection checkbox A/B test

Start an A/B test described in https://github.com/Automattic/wp-calypso/issues/3826. Shows a checkbox for Privacy Protection instead of the two-buttons at the bottom of the form that are shown now.

Original | Variation
------------ | -------------
![privacy_original](https://cloud.githubusercontent.com/assets/3011211/13586826/35a1b4bc-e478-11e5-8989-99364634edfc.png) | ![privacy_checkbox_icon](https://cloud.githubusercontent.com/assets/3011211/13586827/35a4c29c-e478-11e5-8c77-81a47cbf2e69.png)

**Testing**
*the `original` variation*
- Run this in your console: `localStorage.setItem( 'ABTests', '{"privacyCheckbox_20000307":"original"}' );`
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Assert that you can continue to the payment screen with privacy by clicking 'Add Privacy Protection' at the bottom of the page.
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Assert that you can continue to the payment screen with privacy by clicking 'Publish My Information' at the bottom of the page, then 'Add Privacy Protection' in the dialog that appears.
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Assert that you can continue to the payment screen without privacy by clicking 'Publish My Information' at the bottom of the page, then 'Publish My Information' in the dialog that appears.

*the `checkbox` variation`
- Run this in your console: `localStorage.setItem( 'ABTests', '{"privacyCheckbox_20000307":"checkbox"}' );`
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Click 'Continue to Checkout'.
- Assert that the privacy dialog appears.
- Assert that you can continue to the payment screen without privacy by clicking 'Publish My Information' in the dialog.
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Click 'Continue to Checkout'.
- Assert that the privacy dialog appears.
- Assert that you can continue to the payment screen with privacy by clicking 'Add Privacy Protection' in the dialog.
- Visit `/domains/add/:site` and select a domain.
- Enter valid domain details on the domain details screen.
- Check the box next to 'Please keep my information private'.
- Assert that you can continue to the payment screen with privacy by clicking 'Continue to Checkout' at the bottom of the page.
